.yamato/_abv.yml

```diff
@@ -47,17 +37,23 @@ all_project_ci_nightly_trunk:
          rerun: on-new-revision
       -  path: .yamato/all-vfx_urp.yml#Nightly_VFX_URP_trunk
          rerun: on-new-revision
+      -  path: .yamato/all-urp_performance_boatattack.yml#PR_URP_Performance_BoatAttack_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-urp_performance_lostcrypt.yml#PR_URP_Performance_LostCrypt_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-urpupdate.yml#PR_URPUpdate_trunk
+         rerun: on-new-revision
+      -  path: .yamato/all-hdrp_performancetests.yml#PR_HDRP_PerformanceTests_trunk
+         rerun: on-new-revision
+      -  path: .yamato/_projectcontext.yml#all_package_ci_project_trunk
+         rerun: on-new-revision
     triggers:
         recurring:
           -  branch: master
              frequency: 1 * * ?
 all_project_ci_weekly_trunk:
     name: _Weekly ABV against trunk
     dependencies: []
-    triggers:
-        recurring:
-          -  branch: master
-             frequency: weekly
```

Comment:
did this mean to get removed?

Reply:
yep, because weekly is empty anyways. if we want, we can keep the trigger though, I've put the toggles [here ](https://github.com/Unity-Technologies/Graphics/blob/yamato/separate-python/.yamato/config/__shared.metafile#L19) so we can just set it to true. Should I add it or not?

Reply:
ah okay, this is fine then